module-4/romanToDec.js

```diff
@@ -6,4 +6,79 @@
  * @param {string} roman
  * @returns {number} the Arabic (decimal) equivalent of the parameter
  * @throws Error in case of invalid input
- */
\ No newline at end of file
+ */
```

Comment:
That's not exactly how roman numerals work, XVII should be 17 for example, IV=4, etc

Reply:
A new solution has been added.

Reply:
Could you please create a separate test file, based on the other exercises

Reply:
Done